applications/aja_video_capture/cpp/metadata.json

```diff
@@ -25,7 +25,7 @@
 			]
 		},
 		"platforms": ["x86_64", "aarch64"],
-		"tags": ["Video", "AJA"],
+		"tags": ["computer vision and perception", "video", "frame capture pipeline", "AJA holoviz integration", "AJA", "holoviz", "streaming"],
```

Comment:
Would remove "AJA holoviz integration", "holoviz" and "streaming" (all the examples stream)